### PR TITLE
Style/musician profile

### PR DESCRIPTION
**Pull request by oacortez**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- Styled landing page, form, all submissions, and single submission views
- Add animation for all main buttons

What (if anything) did you refactor?
- Added some divs for styling 
- Refactor most of all scss files to make site presentable

Were there any issues that arose?
- It's not dynamic (Site will break at once the page view shrinks)
- Did not refactor css code for Sass due time
- Not really an issue but I did leave commented code which I can clean up later today! 

Is there anything that you need from your teammate?
- Please check testing suites I think some test might fail due to targeting of class names not 100% sure though

Any other comments, questions, or concerns?
- I notice that I was making all these changes under one branch which I am sure it's bad practice. Will do better next time 
- Did not commit frequently and not descriptive will do better on my end

### CODE REVIEW

File: src/components/LandingPage/LandingPage.scss [hunk 1]
```diff
@@ -11,52 +11,134 @@
...
 .winner-container {
-  @include flex(column, center, center);
-  width: 75%;
-  margin: 50px;
+  // @include flex(column, center, center);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    justify-content: space-around;
```
Comment (mr-ryan12) [inline src/components/LandingPage/LandingPage.scss:35]: Just noticed two `justify-content`'s here and curious about the commented out `mixin` above.

File: src/components/LandingPage/LandingPage.scss [hunk 2]
```diff
@@ -11,52 +11,134 @@
...
 }
 
-.submission-dates-may {
-  top: 50%;
+// .greeting-message {
+//   font-size: 1.3rem;
+//   width: 80%;
+//   text-align: center;
+//   color: white;
+// }
```
Comment (mr-ryan12) [inline src/components/LandingPage/LandingPage.scss:67]: Thank you for mentioning in your review to remove the commented out code.

File: src/components/MusicianCard/MusicianCard.scss [hunk 3]
```diff
@@ -2,33 +2,56 @@
 
 .card {
   justify-self: center;
```
Comment (mr-ryan12) [inline src/components/MusicianCard/MusicianCard.scss:4]: Oh cool, never have seen a `justify-self` before. Interesting!

File: src/components/LandingPage/LandingPage.scss [hunk 4]
```diff
@@ -11,52 +11,134 @@
...
+        border-right:7px solid #b94625;
+      }
+      &:hover {
+        border-top-right-radius:0px;
+        border-bottom-left-radius:0px;
+    
+        &:before,&:after {
+         width:100%;
+          height:100%;
+        }
```
Comment (mr-ryan12) [inline src/components/LandingPage/LandingPage.scss:125]: Nice use of pseudo-classes!

**Review by mr-ryan12 — APPROVED** [review]
Hey Oscar! Really looks great! Just wanted to get some clarification on some things:
- The white border at the bottom of the hero image on the landing page. Are you seeing it on your end or is it just me?
- Looks like the submission modal buttons are turquoise. Didn't know if this is by design. If not, it's an easy fix.
- Thank you for mentioning removing the commented out code.
- Turquoise background when hamburger menu collapses. Again, easy fix if not intentional.
- Just wanted to see where we are with responsive design and refactoring with SASS.

I'm sure we can address this at our meeting in a couple of minutes, but put this here for an easy reference. Sorry to sound so nit-picky, but these were just a few things I noticed. 
Thank you again for making our app so stylish!

**Review by Echory — APPROVED** [review]
Thank you for styling all of this, it looks great

**Comment by aliroemhildt** [comment]
Closing this PR due to failing tests

**Pull request closed without merging**